openshift/openshift-ansible-contrib
# Configure different Docker volume sizes for different roles

#### What does this PR do?
User can now specify different Docker volume size for the servers.
Additionally
- I changed `app_volume_size` to `node_volume_size` as `node` is used to describe an application node in multiple places.
- I added `lb_volume_size` instead of a hard-coded value in `heat_stack.yaml.j2`.

#### How should this be manually tested?
1) In `<path to inventory>/group_vars/all.yml`, uncomment line(s) with variables for specifying Docker volume size (`docker_<role>_volume_size`) and change its value to a valid volume size.
2) Run the provisioning.
3) Display all servers with `openstack server list`. Choose one server with a default volume size and one with a changed size, not their ID.
4) Display server's volume size by running `openstack server show <server ID>`, note the volume ID (last line) and then run `openstack volume show <volume ID>`. Check that the right values were used.

#### Is there a relevant Issue open for this?
--

#### Who would you like to review this?
cc: @tomassedovic @bogdando PTAL
